### PR TITLE
Remove coming soon, add nornir plugin

### PR DESCRIPTION
**Pull request by itdependsnetworks**

### CODE REVIEW

**Comment by itdependsnetworks** [comment]
Opening up the index file locally, I get 
image

**Review by cmsirbu — CHANGES_REQUESTED** [review]
Hey, thanks for the PR... would you please remove the "Network to Code Nautobot Apps" link as it's redundant with the big yellow button? I left it there until plugin-nornir was ready to go in (it's fine where it is now).

This also solves the need to have an empty row in the left hand side.

File: html/index.html [hunk 1]
```diff
@@ -169,8 +175,10 @@ <h2 class="my-0 fw-semibold">Nautobot Apps</h2>
                             <li class="list-group-item"><a href="https://nautobot-plugin-ssot.readthedocs.io/en/latest/"
                                     class="link-dark">Nautobot Single Source of Truth
                                     (SSoT)</a><span class="badge text-bg-dark rounded-pill ms-1">App</span></li>
-                            <li class="list-group-item">Nautobot Device Lifecycle Management<span
-                                    class="badge text-bg-dark rounded-pill ms-1">Coming soon</span></li>
+                            <li class="list-group-item"><a
+                                    href="https://github.com/nautobot/nautobot-plugin-device-lifecycle-mgmt/tree/develop/docs"
+                                    class="link-dark">Nautobot Device Lifecycle Management</a><span 
+                                    class="badge text-bg-dark rounded-pill ms-1">App</span></li>
                             <!-- <li class="list-group-item"><a
```
Comment (cmsirbu) [inline html/index.html]: Please remove the commented section.

File: html/index.html [hunk 2]
```diff
@@ -159,8 +162,11 @@ <h2 class="my-0 fw-semibold">Nautobot Apps</h2>
...
                                     class="link-dark">Nautobot Golden Configuration</a><span
                                     class="badge text-bg-dark rounded-pill ms-1">App</span></li>
-                            <li class="list-group-item">Nautobot Device Onboarding<span
-                                    class="badge text-bg-dark rounded-pill ms-1">Coming soon</span></li>
+                            <li class="list-group-item"><a
+                                    href="https://github.com/nautobot/nautobot-plugin-device-onboarding/tree/develop/docs"
+                                    class="link-dark">Nautobot Device Onboarding</a><span
+                                    class="badge text-bg-dark rounded-pill ms-1">App</span>
+                            </li>
                             <!-- <li class="list-group-item"><a
```
Comment (cmsirbu) [inline html/index.html]: Please remove the commented section.

**Review by cmsirbu — CHANGES_REQUESTED** [review]

**Comment by itdependsnetworks** [comment]
Updated view

**Comment by itdependsnetworks** [comment]
Updated view again
image

File: html/index.html [hunk 3]
```diff
@@ -144,45 +144,50 @@ <h2 class="my-0 fw-semibold">Nautobot Apps</h2>
                                 Development of each App.</p>
                         </div>
                         <ul class="list-group list-group-flush">
-                            <li class="list-group-item"><a href="https://docs.nautobot.com/projects/core/en/stable/apps/"
-                                    class="w-100 btn btn-lg btn-warning" target="_blank">To the App Docs!</a>
+                            <li class="list-group-item"><a href="https://www.networktocode.com/nautobot/apps/"
```
Comment (cmsirbu) [inline html/index.html]: This is a docs site and we should be linking to docs - I don't agree with this link change here. The main button sending ppl to the main website (that doesn't link to docs) is not the way. The https://docs.nautobot.com/projects/core/en/stable/apps/ page links to the ecosystem page right at the top if people want to discover more.

**Review by cmsirbu — APPROVED** [review]
LGTM, thank you!

**Pull request merged**